pecos/xmc/base.py

```diff
@@ -214,6 +213,29 @@ def convert_codes_to_csc_matrix(codes, depth):
         )
         return C
 
+    @staticmethod
+    def get_depth(nr_instances, max_leaf_size):
```

Comment:
Please add docstring.

Reply:
also the `depth` concept here is the binary tree depth rather than the HLT depth. We should distinguish them by naming.